macrofusion.py

```diff
@@ -982,7 +1059,7 @@ def __init__(self, name, img_list, img_list_aligned, issend):
         self.img_list_aligned = img_list_aligned
         self.command_fuse  = [data.get_all_settings["enfuser"], "-o", self.name] + data.get_enfuse_options + self.img_list_aligned
         self.command_align = ["align_image_stack", '-a', os.path.join(settings["preview_folder"], settings["align_prefix"])] + data.get_align_options() + self.img_list
-
+        #sprint(' '.join(self.command_align ) + "\n" + ' '.join(self.command_fuse) + "\n")
```

Comment:
Looks like debugging left-over ;)
Also: this information is already printed out in the `run()` method below